docs/hpc/about/pricing-compute.md

```diff
@@ -1,33 +1,53 @@
 # Pricing for Extra Compute
 
-_Example specifications and pricing as of February 2021_
+_Example specifications and pricing as of May 2023_
 
+Please note: You cannot be charged for the hardware until it has been received and invoiced. Since lead times can be extensive, please plan as far ahead as you can. 
 
 ## Pricing for prioritized compute ("CPU")
 
-Cluster compute nodes are purchased by Wynton in units which contain 4 Compute Nodes in a single chassis. Partial nodes can be purchased and contributions will be pooled towards a full system.
+Cluster compute nodes are purchased by Wynton in units that have 4 Compute Nodes in a single chassis. If partial nodes are requested, contributions will be pooled with other requests towards a full system. Another option is to pay $170/per member.q slot/share. 
 
-* CPU/node: 2 × Intel Xeon Gold 6336Y 2.4 GHz 24-core processors
-* RAM/node: 256 GiB
+**Price per 4 compute node unit: ~$33k USD excluding tax (for 4 nodes in a 2U chassis)**
 
-**Price per 4 compute node unit: 33,405 USD (for 4 nodes in a 2U chassis)**
+SMC RM224Q 2U Quad-Node Server 
+Four Hot-Pluggable Compute Modules, each Node Includes:  
+CPU: (2) Xeon Gold 6336Y 2.4GHz Twenty-Four-core  
+Memory: 256GB DDR4-3200 ECC (16x 16GB)  
```

Comment:
Note how we used "RAM" for "Memory" previously.  Also, we use "GiB" instead of "GB", because they're not the same.  I can fix this too.

Reply:
I copied that info from the AHPC quote. I agree it makes sense to make it less wordy, but if it's GB in the quote's product description, then maybe it was a mistake to have GB before?

Reply:
I'm quite sure AHPC quote is uses the wrong unit.  RAM is in GiB and disk storage, typically, in GB.  They differ quite a bit, e.g. 256 GiB = 256*1024^3 = 274,877,906,944 bytes >> 256 GB = 256*1000^3 = 256,000,000,000 bytes.  I know lots of people don't care, but I think we should be correct when we can.